Polyfills/Window/Source/Window.cpp

```diff
@@ -26,6 +26,11 @@ namespace Babylon::Polyfills::Internal
         auto jsNative = global.Get(JsRuntime::JS_NATIVE_NAME).As<Napi::Object>();
         auto jsWindow = constructor.New({});
 
+        // Need a reference or It's destroyed when loading babylon.material.js
+        // TODO: Find why
+        napi_ref result;
+        napi_create_reference(env, jsWindow, 1, &result);
```

Comment:
I filled an issue for this one : https://github.com/BabylonJS/BabylonNative/issues/236